misc.Rmd

```diff
@@ -65,69 +80,45 @@ citation()
 Calling it with a package name tells you how to cite that package:
 
 ```{r}
-citation("lubridate")
+citation("tidyverse")
 ```
 
-To customise the citation for your package, add a `inst/CITATION` that looks like this:
+The associated `inst/CITATION` file looks like this:
 
 ```{r, echo = FALSE, comment = ""}
-citation <- readLines(system.file("CITATION", package = "lubridate"))
+citation <- readLines(system.file("CITATION", package = "tidyverse"))
 cat(citation, sep = "\n")
 ```
 
-You need to create `inst/CITATION`.
-As you can see, it's pretty simple: you only need to learn one new function, `citEntry()`.
-The most important arguments are:
-
--   `entry`: the type of citation, "Article", "Book", "PhDThesis" etc.
-
--   The standard bibliographic information like `title`, `author` (which should be a `personList()`), `year`, `journal`, `volume`, `issue`, `pages`, ...
-
-A complete list of arguments can be found in `?bibentry`.
-
-Use `citHeader()` and `citFooter()` to add additional exhortations.
-
-### Other languages {#sec-inst-other-langs}
-
-Sometimes a package contains useful supplementary scripts in other programming languages.
-Generally, you should avoid these, because it adds an additional extra dependency, but it may be useful when wrapping substantial amounts of code from another language.
-For example, [gdata](https://cran.r-project.org/web/packages/gdata/index.html) wraps the Perl module [Spreadsheet::ParseExcel](https://search.cpan.org/~dougw/Spreadsheet-ParseExcel-0.65/) to read excel files into R.
-
-The convention is to put scripts of this nature into a subdirectory of `inst/`, `inst/python`, `inst/perl`, `inst/ruby` etc.
-If these scripts are essential to your package, make sure you also add the appropriate programming language to the `SystemRequirements` field in the `DESCRIPTION`.
-(This field is for human reading so don't worry about exactly how you specify it.)
+You can call `usethis::use_citation()` to initiate this file and fill in your details.
```

Comment:
I tried to word this in a way that will be robust to changes we might make to `use_citation()`.